index.html

```diff
@@ -0,0 +1,188 @@
+                
+            </div>
+            <div class="item-menu" id="logo-container">
+            
+            </div>
+            <div class="item-menu" id="search-microfone">
+                <div id="search-contain"><input type="search" name="" id="search"></div>
+                <div id="submit"> <img src="css/img/lupa.png" > </div>
+                <div id="microfone"> <img src="css/img/microfono.png" > </div>
+            </div>
+            <div class="item-menu" id="right-icons">
+                
+            </div>
+    </nav>
+    <section id="video-section">
+        <div id="video-contain">
+            <iframe  src="https://www.youtube.com/embed/V74l_zS1x8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>        
+          <h2 id="video-name"> Contributing to Open source Part 1: The Easy Way</h2>
+            <div id="video-bar">
+                <img id="channel-img" src="css/img/channel.jpg" >
+                <div id="channel">
+                    <h3 id="channel-name">The  Odin Project</h3>
+                    <h5 id="subs"> 1.12 K subscribers</h5>
+                </div>
+                <div id="subscribe">
+                    <h3>  Subscribe</h3>
+            </div>
+  
+                <div id="likes" class="video-bar-icons">  
+                    <img  id="like-img"  src="css/img/like.png">                  
+                    <h3 class="video-icons-text">499</h3>
+                    <h3 class="video-icons-text">|</h3>
+                    <img id="unlike-img" src="css/img/unlike.png" >
+                </div>
+       
+                <div id="shares" class="video-bar-icons">
+                    <img  id="share-img" src="css/img/share.png" alt="">
+                    <h3  class="video-icons-text">Share</h3>
+                </div>
+                <div id="dowloads" class="video-bar-icons">
+                    <img  id="download-img" src="css/img/download.png" alt="">
+                    <h3  class="video-icons-text">Download</h3>
+                </div>
+
+                <div id="more-options">
+                    <h3>. . .</h3>
+                </div>
+
+            </div>
+            <div id="video-description">
+                <h5>How do you contribute to open source?  In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github.  I'll also describe what kinds of contributions open source software projects are typically looking for.  As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum for learning web development with Ruby on Rails.  You can find more information about that project at h</h5>

```

Comment:
Apply the same here

Reply:
- [ ] Great effort making use of inline comment, but as part of this milestone I would kindly suggest you give the reason `WHY` you are requesting for a review.
- [ ] Also I would suggest you make your review clear and easy to understand by the student.
- [ ] kindly give a suggestion when requesting for changes rather than given an order.

Review guideline :point_down: 
1. The student always includes an explanation, a "why", for each requested change. For example:
      - This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
      - This request lacks a "why": Why don't you use an article tag here instead of a div?

2. The student comments are clear and easy to understand. For Example:
      - This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
      - This is a vague comment: The README is not descriptive, why don't you make it better?

3. The student phrased the comments politely, as suggestions instead of orders.
For example:
      - This is a polite suggestion: "I think it would be better to use semantic tags"
      - This reads as an order: "Use semantic tags"
> N.B: kindly edit the review and make the changes, you don't need to create a new inline comment, just simply edit the ones you have given before and make it look perfect before you resubmit for review. Kindly add it to all the comments you have initially.
Thank you.